src/widgets/DataTable/TouchableNoFeedback.js

```diff
@@ -24,10 +24,11 @@ const TouchableNoFeedback = ({ children, style, ...touchableProps }) => (
 
 TouchableNoFeedback.defaultProps = {
   style: null,
+  children: null,
 };
 
 TouchableNoFeedback.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
```

Comment:
Just getting a warning if null is passed as a child. This is pretty inconsequential